src/test/java/org/openmrs/contrib/qaframework/automation/AllergiesSteps.java

```diff
@@ -0,0 +1,6 @@
+package org.openmrs.contrib.qaframework.automation;
+
+
+public class AllergiesSteps {
+	
```

Comment:
this should be out of this change and rather for a separate ticket

Reply:
Thank you @k-joseph I have made the changes